src/app/sidenav/sidenav.component.scss

```diff
@@ -0,0 +1,8 @@
+.mat-icon {
```

Comment:
This can be removed with the new structure

Reply:
The star icon is vertically misaligned (it's 4px too low). The styling:
```
.mat-icon {
    margin-bottom: 4px;
}
```
is just to make sure the icon is aligned on the same level as the community name.

Reply:
Hmm, strange,

I removed the margin-bottom styling and measured the height of the icon and it is perfectly aligned vertically when counting the pixels.